trade-finance-manager-ui/server/types/data-transfer-objects/get-deal-response.ts

```diff
@@ -0,0 +1,9 @@
+import { DealSubmissionType } from '@ukef/dtfs2-common';
+
+/**
+ * Data Transfer Object sent from the trade-finance-manager-api endpoint /v1/deals/:id
+ *
+ * This type is likely incomplete and should be added
+ * to as and when new properties are discovered
+ */
+export type GetDealResponse = { dealSnapshot: { details: { ukefDealId: string }; submissionType: DealSubmissionType } } | undefined;
```

Comment:
Can it be multiline please to improve readability.
Is this union type because this DTO can also be used when deal is not found?

Reply:
Yep, this is what `getDeal` returns from the api.
I'm not sure if `response?.data` will ever be undefined, but it seems safest to assume that it could be